config/registry/schemas/common-types/cidr-block.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$id": "https://app.runx.dev/common-types/cidr-block",
```

Comment:
why is this needed?

Reply:
These are used when schemas reference each other